src/TransactionalSession.StorageTable.AcceptanceTests/ConfigureAzureTablePersistence.cs

```diff
@@ -17,7 +18,8 @@ public Task Configure(string endpointName, EndpointConfiguration configuration,
 
         sagaPersistence.Compatibility().DisableSecondaryKeyLookupForSagasCorrelatedByProperties();
 
-        configuration.UsePersistence<AzureTablePersistence, StorageType.Outbox>();
+        var outboxPersistence = configuration.UsePersistence<AzureTablePersistence, StorageType.Outbox>();
```

Comment:
Why not `EnableOutbox`?

Reply:
At the time we have chosen this way of setting up the storages in this repo because the saga persister has a bunch of defaults that would interfere with things. I did stick to that for now even though you could argue the tests in here do not require sagas. 

Ranting a bit: I think the defaults could have been flipped for the core v8 version if we actually did have mechanics to think ahead like that ;)